### DISCUSSION TITLE
Possible to isolate the import to a specific team?

### DISCUSSION THREAD

**Discussion by gbrits** [simonhamp/laravel-nova-csv-import#58] [Q&A] [answered]
First - apologies for posting an issue, I couldn't find discussions or any other way to pose the question... secondly, awesome package - I would really love to be able to use it for my specific use case.

The platform that requires importing has tenancy - so multiple teams and each user has a current_team_id - the usual Jetstream implementation. All Resources have a `team_id` on them - is it possible to restrict importing to a specific team so that this tool can be made useful to individual teams? Perhaps a way to autopopulate `team_id` while importing? Thanks!

**Comment by HeadStudios** [comment]
Dude I think this sub is dead - it's a great plugin and works for some models but not others - if the creator is there we need you!

**Comment by HeadStudios** [comment]
Btw I'm looking through the source code now - I'm checking out your question - I'm sure you can check for the loggeg in user with a specific meta and then... you know... import based on that paramater.

**Comment by matt7ds** [comment, marked as answer]
Have you thought about:

- publishing the csv-import config file
- Pointing to your own Importer class
- Extending the standard Importer class and putting the logic you need in the model() function of that class (which is the part that fills each row of the model from the supplied data)

**Comment by simonhamp** [comment]
@gbrits as @matt7ds suggests, this would be your best approach.

>I couldn't find discussions or any other way to pose the question

I will open GitHub Discussions so that this can be used for these kinds of questions/discussions.

**Comment by simonhamp** [comment]
>if the creator is there we need you!

@HeadStudios I'm here. Unfortunately, I don't earn any money off of this package just yet, so I'm not able to prioritise support. I will get around to it eventually.

If you need urgent help though, I'm happy to discuss booking some time to work on this for you.
